packages/core/src/components/StepList/StepList.example.jsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/display-name */
+import React from 'react';
+import StepList from './StepList';
+
+export default function() {
+  return (
+    <div style={{ maxWidth: '628px' }}>
+      <StepList
+        inverse
```

Comment:
`inverse` probably shouldn't be here (it's not a supported prop)

Reply:
whoops

Reply:
left over from early tests before talking to Cindy about inverse support